src/cmd_handlers.c

```diff
@@ -1011,6 +1016,44 @@ goto_cmd(const cmd_info_t *cmd_info)
 	return 0;
 }
 
+static int
```

Comment:
Please add a short comment above like `/* Opens an editor to edit register contents. */`.